app/src/display/Kconfig

```diff
@@ -29,6 +29,29 @@ config ZMK_DISPLAY_STATUS_SCREEN_CUSTOM
 
 endchoice
 
+choice ZMK_DISPLAY_WORK_QUEUE
+    prompt "Work queue selection for UI updates"
```

Comment:
Should we default this one way or the other? I'm not sure if most displays update quickly or slowly.

Reply:
So, the default, if not set, is the first defined in the choice. By not *explicitly* setting a default, then it's easier to default it in `Kconfig.defconfig` files for shields, etc.

So, the *implicit* default is the system work queue, which I think is reasonable.

Reply:
Ah. I didn't realize it worked that way. (Doesn't help that I cannot find a single easy to understand resource on how kconfig works.)